### PR TITLE
Dpp 1097 select a transfer connection or continue in normal flow

### PR DESCRIPTION
**Pull request by Pauljgraham**
Handle the case that a TransferConnection (a Funding Transfer Sender has been selected)

### CODE REVIEW

File: src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs [hunk 1]
```diff
@@ -1075,6 +1099,22 @@ private Task<OrchestratorResponse<ConfirmProviderViewModel>> GetProvider(string
...
 
+        private async Task<(long?, string)> GetTransferConnectionInfo(string hashedAccountId, string transferConnectionCode, string externalUserId)
+        {
+            string transferSenderName = null;
+            long? transferSenderId = null;
+
+            if (!string.IsNullOrEmpty(transferConnectionCode))
+            {
+                var transferConnections = await GetTransferConnectionsNoAuthorizationCheck(hashedAccountId, externalUserId);
+                var transferConnection = transferConnections.TransferConnections.Single(x =>
```
Comment (chrisfoster76) [inline src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs]: This throws an exception if the user has attempted to tamper with values submitted earlier in the multi-stage journey by editing a hidden field. While we don't care about being graceful for such users, we might want to consider throwing a more appropriate exception (argument out of range? invalid operation? I don't know, but something) and a little message in the logs that makes sense for us when we see it. Maybe think about what log level is appropriate (do we want to be warned about this?).

Comment (chrisfoster76) [reply src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs]: Or maybe that's total overkill for something that will likely never happen!

Comment (Pauljgraham) [reply src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs]: I know what you mean. But we do the same with Legal Entities.

**Comment by satya-sanikommu** [comment]
Testing completed, using AccountsApi response stub, No regression issues. Ready to merge

**Pull request merged**